Src/Plugins/Tests.Security/Tests.Security.csproj

```diff
@@ -13,6 +13,10 @@
     </EmbeddedResource>
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="TestData\SecurePlaintextSecrets\ExpectedOutputs\SEC101_114.PostgreSqlConnectionString.sarif" />
+  </ItemGroup>
```

Comment:
u can remove this

Reply:
Removed

---
In reply to: [568862985](https://github.com/microsoft/sarif-pattern-matcher/pull/157#discussion_r568862985) [](ancestors = 568862985)